### PR TITLE
[zh-cn]: create the translation of `declarativeNetRequest.DYNAMIC_RULESET_ID` property

### PR DESCRIPTION
**Pull request by tianyeeT**
…ESET_ID` property

### CODE REVIEW

File: files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/dynamic_ruleset_id/index.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+---
+title: declarativeNetRequest.DYNAMIC_RULESET_ID
+slug: Mozilla/Add-ons/WebExtensions/API/declarativeNetRequest/DYNAMIC_RULESET_ID
+l10n:
+  sourceCommit: 43e3ff826b7b755b05986c99ada75635c01c187c
+---
+
+{{AddonSidebar}}
+
+通过使用 {{WebExtAPIRef("declarativeNetRequest.updateDynamicRules","updateDynamicRules")}} 添加到扩展的动态规则集的规则集 ID。
```
Comment (yin1999) [inline files/zh-cn/mozilla/add-ons/webextensions/api/declarativenetrequest/dynamic_ruleset_id/index.md:10]:
```suggestion
通过使用 {{WebExtAPIRef("declarativeNetRequest.updateDynamicRules","updateDynamicRules")}} 添加到扩展的动态规则中的规则集 ID。
```

**Review by yin1999 — APPROVED** [review]

**Review by judyburgett3 — COMMENTED** [review]
Thank you

**Review by judyburgett3 — COMMENTED** [review]
Yes approve this

**Review by judyburgett3 — COMMENTED** [review]
Ok

**Pull request merged**